.run/Run all clients.run.xml

```diff
@@ -0,0 +1,7 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Run all clients" type="CompoundRunConfigurationType">
+    <toRun name="Banking Client 2" type="Application" />
```

Comment:
Ok, isso é meio inútil, mas colocar isso em ordem crescente.

Reply:
Não faz diferença, ele roda tudo fora de ordem mesmo